src/main/runner.ts

```diff
@@ -149,6 +149,7 @@ export default class RunnerMain extends Runner {
   private emitSubprocessEvents() {
     for (const { event, data, type } of this.subprocessTestResults.events) {
       if (type === 'runner') {
+        assert(event);
```

Comment:
This looks like a debug code

Reply:
It's just to ensure event exists. It might become undefined if a bug is introduced elsewhere.

Reply:
Looks like a premature optimization. Let's either add a test for it or remove this.